docs/fundamentals/code-analysis/quality-rules/performance-warnings.md

```diff
@@ -71,3 +71,4 @@ Performance rules support high-performance libraries and applications.
 | [CA1860: Avoid using 'Enumerable.Any()' extension method](ca1860.md) | It's more efficient and clearer to use `Length`, `Count`, or `IsEmpty` (if possible) than to call <xref:System.Linq.Enumerable.Any%2A?displayProperty=nameWithType> to determine whether a collection type has any elements. |
 | [CA1861: Avoid constant arrays as arguments](ca1861.md) | Constant arrays passed as arguments are not reused which implies a performance overhead. Consider extracting them to 'static readonly' fields to improve performance. |
 | [CA1864: Prefer the 'IDictionary.TryAdd(TKey, TValue)' method](ca1864.md) | Both <xref:System.Collections.Generic.Dictionary%602.ContainsKey(%600)?displayProperty=nameWithType> and <xref:System.Collections.Generic.Dictionary%602.Add%2A?displayProperty=nameWithType> perform a lookup, which is redundant. It's is more efficient to call <xref:System.Collections.Generic.Dictionary%602.TryAdd%2A?displayProperty=nameWithType>, which returns a `bool` indicating if the value was added or not. `TryAdd` doesn't overwrite the key's value if the key is already present. |
+| [CA1865: Unnecessary call to 'Contains' for sets](ca1865.md) | Both <xref:System.Collections.Generic.ISet%601.Add%2A> and <xref:System.Collections.Generic.ICollection%601.Remove%2A> perform a lookup which makes calling <xref:System.Collections.Generic.ICollection%601.Contains%2A> before redundant. It is more efficient to call <xref:System.Collections.Generic.ISet%601.Add%2A> and <xref:System.Collections.Generic.ICollection%601.Remove%2A> directly, which will return a Boolean value indicating whether an item has been added or removed. |
```

Comment:
Please update to match suggestion from main file.

Reply:
Updated in [e055b1c](https://github.com/dotnet/docs/pull/36379/commits/e055b1cae46fe07e6c02527aff246dce1e4ada9c).